### PR TITLE
added location service dialog box for Android

### PR DESCRIPTION
**Pull request by ochanje210**

### CODE REVIEW

File: shared/components/map.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ import eventClickPng from '../resources/marker/event_big.png';
 import facilityClickPng from '../resources/marker/facility_big.png';
 import warningClickPng from '../resources/marker/warning_big.png';
 import {API_GEODATA, API_KEY} from '../utils';
+import LocationServicesDialogBox from 'react-native-android-location-services-dialog-box';
```
Comment (yuda1124) [inline shared/components/map.js]: is it for android only?

**Comment by yuda1124** [comment]
LGTM

File: shared/components/map.js (hunk 1 above)
Comment (ochanje210) [inline shared/components/map.js]: we do not need another module for IOS since IOS have default location service dialog box.

Comment (ochanje210) [inline shared/components/map.js]: and yes it is android only

**Comment by ochanje210** [comment]
lgtm

**Pull request merged**